runtime/vm/MHInterpreter.inc

```diff
@@ -2373,7 +2373,12 @@ VM_MHInterpreter::nativeMethodHandleCallout(void *function, UDATA *javaArgs, U_8
 		IDATA offset = 0;
 		for (U_8 i = 0; i < argTypeCount; i++) {
 			bool isStructArg = false;
-			if (FFI_TYPE_STRUCT == args[i+1]->type) {
+			if (FFI_TYPE_STRUCT == args[i+1]->type
```

Comment:
Don't modify any code in `MHInterpreter.inc` as it belongs to the outdated OpenJ9 MH which is totally replaced by OpenJDK MH.

Reply:
Removed changes from MHInterpreter.inv in https://github.com/eclipse-openj9/openj9/pull/19645/commits/1e752555416853814f4beb0cf0eea5f3fe8580bf